Update properties panel examples to the new API

__What should we do?__

Update examples to account for breaking changes in https://github.com/bpmn-io/properties-panel/pull/134

* [ ] Update https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel-extension
* [ ] Update https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel-async-extension


__Why should we do it?__

The examples should be up to date.

---

Related to https://github.com/bpmn-io/bpmn-js-examples/issues/135

When I try to run `npm install` in [properties-panel-extension](https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel-extension) I get the following error:
```
npm ERR! Could not resolve dependency:
npm ERR! peer @bpmn-io/properties-panel@"0.11.x" from bpmn-js-properties-panel@1.0.0-alpha.5
npm ERR! node_modules/bpmn-js-properties-panel
npm ERR!   bpmn-js-properties-panel@"^1.0.0-alpha.0" from the root project
```
Am I correct in assuming a relation to this issue? Or is that a separate problem that I should report?

> When I try to run `npm install` in [properties-panel-extension](https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel-extension) I get the following error:
> 
> ```
> npm ERR! Could not resolve dependency:
> npm ERR! peer @bpmn-io/properties-panel@"0.11.x" from bpmn-js-properties-panel@1.0.0-alpha.5
> npm ERR! node_modules/bpmn-js-properties-panel
> npm ERR!   bpmn-js-properties-panel@"^1.0.0-alpha.0" from the root project
> ```
> 
> Am I correct in assuming a relation to this issue? Or is that a separate problem that I should report?

Hi @Adrian-Bielefeldt ,

thanks for spotting! Could you please report a seperate issue? I think this is something we can/should fix even before doing this issue here.

Thanks!

--- a/properties-panel-async-extension/app/provider/magic/parts/SpellProps.js
+++ b/properties-panel-async-extension/app/provider/magic/parts/SpellProps.js
@@ -9,7 +9,8 @@ export default function(element) {
   return [
     {
       id: 'spell',
-      component: <Spell id="spell" element={ element } />,
+      element,
+      component: Spell,
       isEdited: isSelectEntryEdited
     }
   ];

--- a/properties-panel-async-extension/package.json
+++ b/properties-panel-async-extension/package.json
@@ -46,9 +46,9 @@
     "webpack-cli": "^4.9.1"
   },
   "dependencies": {
-    "@bpmn-io/properties-panel": "^0.10.0",
+    "@bpmn-io/properties-panel": "^0.11.0",
     "bpmn-js": "^9.0.2",
-    "bpmn-js-properties-panel": "^1.0.0-alpha.0",
+    "bpmn-js-properties-panel": "^1.0.0-alpha.5",
     "cors": "^2.8.5",
     "express": "^4.17.2",
     "jquery": "^3.5.1",

--- a/properties-panel-extension/README.md
+++ b/properties-panel-extension/README.md
@@ -85,7 +85,7 @@ function createMagicGroup(element, translate) {
   const magicGroup = {
     id: 'magic',
     label: translate('Magic properties'),
-    entries: spellProps(element, translate)
+    entries: spellProps(element)
   };
 
   return magicGroup
@@ -101,19 +101,20 @@ The "spell" entry is defined in [`SpellProps`](app/provider/magic/parts/SpellPro
 import { TextFieldEntry, isTextFieldEntryEdited } from '@bpmn-io/properties-panel';
 import { useService } from 'bpmn-js-properties-panel'
 
-export default function(element, translate) {
+export default function(element) {
 
   return [
     {
       id: 'spell',
-      component: <Spell element={ element } translate={ translate } />,
+      element,
+      component: Spell,
       isEdited: isTextFieldEntryEdited
     }
   ];
 }
 
 function Spell(props) {
-  const { element } = props;
+  const { element, id } = props;
 
   const modeling = useService('modeling');
   const translate = useService('translate');
@@ -130,6 +131,7 @@ function Spell(props) {
   }
 
   return <TextFieldEntry
+    id={ id }
     element={ element }
     description={ translate('Apply a black magic spell') }
     label={ translate('Spell') }

--- a/properties-panel-extension/app/provider/magic/parts/SpellProps.js
+++ b/properties-panel-extension/app/provider/magic/parts/SpellProps.js
@@ -1,12 +1,13 @@
 import { TextFieldEntry, isTextFieldEntryEdited } from '@bpmn-io/properties-panel';
-import { useService } from 'bpmn-js-properties-panel'
+import { useService } from 'bpmn-js-properties-panel';
 
 export default function(element) {
 
   return [
     {
       id: 'spell',
-      component: <Spell id="spell" element={ element } />,
+      element,
+      component: Spell,
       isEdited: isTextFieldEntryEdited
     }
   ];

--- a/properties-panel-extension/package.json
+++ b/properties-panel-extension/package.json
@@ -46,9 +46,9 @@
     "webpack-cli": "^4.9.1"
   },
   "dependencies": {
-    "@bpmn-io/properties-panel": "^0.10.0",
+    "@bpmn-io/properties-panel": "^0.11.0",
     "bpmn-js": "^9.0.2",
-    "bpmn-js-properties-panel": "^1.0.0-alpha.0",
+    "bpmn-js-properties-panel": "^1.0.0-alpha.5",
     "jquery": "^3.5.1",
     "min-dash": "^3.7.0"
   }

--- a/properties-panel/package.json
+++ b/properties-panel/package.json
@@ -34,9 +34,9 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@bpmn-io/properties-panel": "^0.10.0",
+    "@bpmn-io/properties-panel": "^0.11.0",
     "bpmn-js": "^9.0.2",
-    "bpmn-js-properties-panel": "^1.0.0-alpha.0",
+    "bpmn-js-properties-panel": "^1.0.0-alpha.5",
     "jquery": "^3.5.1",
     "min-dash": "^3.7.0"
   },

